der/derive/src/attributes.rs

```diff
@@ -79,6 +79,9 @@ pub(crate) struct FieldAttrs {
 
     /// Is the inner type constructed?
     pub constructed: bool,
+
+    /// Is the type skipped?
+    pub skipped: Option<Path>,
```

Comment:
if this is all to handle `PhantomData`, what if instead we just impl'd the relevant `der` traits on `PhatomData` as no-ops?

Reply:
Yes.
I guess that could work.

Reply:
Except I can't implement `der::Tagged` ?

Reply:
I mean, this has to be handled in the `Sequence` implementation body, and thus has to be handled in the derive. I am confusing myself here?